src/App.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { BrowserRouter, Route, Routes } from 'react-router-dom';
+import './App.css';
+import Calculator from './components/Calculator';
+import Navbar from './components/Navbar';
+import Home from './pages/Home';
+import Quote from './pages/Quote';
+
+function App() {
+  return (
+    <>
+      <BrowserRouter>
+        <Navbar />
+
+        <Routes>
+          <Route exact path="/" element={<Home />} />
+          <Route exact path="/calculator/*" element={<Calculator />} />
+          <Route exact path="/quote/*" element={<Quote />} />
+        </Routes>
+      </BrowserRouter>
+    </>
+  );
+}
```

Comment:
- [ Required ] Good job implementing the `Math Magicians: full website` however, I noticed that your design doesn't match the wireframe provided.
  
  - The header should have the text `Math Magicians` aligned to the left of the header
  - The menu should be aligned to the right of the header
  - The calculator should be aligned to the left of the page

Please see the screenshot below for more details on this and rectify.

## Wireframe
![Screenshot from 2023-02-01 12-43-55](https://user-images.githubusercontent.com/16392046/216011549-73bc6ce5-fe18-4614-b606-97ffa8299962.png)

## Your design
![Screenshot from 2023-02-01 13-01-22](https://user-images.githubusercontent.com/16392046/216012160-3f819f3c-0067-4fed-bea2-7e10d52dbc01.png)